### PR TITLE
Add BG Client card in Learn more section

### PR DESCRIPTION
**Pull request by andrealbiac**
First PR with the new site updates 👾

Starting with the easy ones,
- I uploaded the new assets that we'll need for further tweaks
- And updated the BG Client card in Learn more section as it follows:

Captura de pantalla 2024-09-25 a las 14 11 02

Let me know if I missed/ should change anything for future PRs!
Cheers

### CODE REVIEW

**Comment by andrealbiac** [comment]
Sorry, I closed the PR accidentally by changing the name of the branch, my bad. Another beginner mistake here 😅 Reopened it now.

File: packages/nextjs/pages/index.tsx [hunk 1]
```diff
@@ -328,15 +328,16 @@ const Home: NextPage<{
...
     props: {
-      stats,
+      stats: {
+        builderCount: "50",
+        buildCount: "100",
+        streamedEth: 100,
+        buildersIncrementMonth: 10,
+        buildsIncrementMonth: 10,
+        streamedEthIncrement: 10,
+      },
```
Comment (Pabl0cks) [inline packages/nextjs/pages/index.tsx:340]: I suppose this change was for your internal tests? We'd destroy the stats if we merge :)

**Review by Pabl0cks — CHANGES_REQUESTED** [review]
Looking great to me @andrealbiac!! 🙌

Only left a comment that needs to be checked before merging, rest feels good, TYSM 🙏

**Comment by andrealbiac** [comment]
Hey! Closing this PR to make two separate ones starting from scratch. Addressing Pablo's message in Telegram!

**Pull request closed without merging**